### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by MoneyDropLobby**
Added override keyword to methods class. The override keyword helps catch mistakes (like misspelling the method's name in the derived class or mismatching the method's signatures) at compile time

### CODE REVIEW

File: src/backend/reactions/reaction.cpp [hunk 1]
```diff
@@ -64,28 +64,31 @@ namespace big
...
+            if (g_hooking->get_original<hooks::send_chat_message>()(
+                *g_pointers->m_gta.m_send_chat_ptr,
+                g_player_service->get_self()->get_net_data(),
+                chat.c_str(),
+                false))
+            {
+                notify::draw_chat(chat, g_player_service->get_self()->get_name(), false);
+            }
+        });
+    	}
```
Comment (Yimura) [inline src/backend/reactions/reaction.cpp]: Indentation is off.

File: src/backend/reactions/interloper_reaction.cpp [hunk 2]
```diff
@@ -24,28 +24,33 @@ namespace big
...
-			g_notification_service->push_warning("Protections", notification);
+			std::string notification = std::format("{} {} {}",
+				m_notify_message,
+				attacker->get_name(),
+				victim->get_name());
+			g_notification_service->push_warning("Protections", notification.c_str());
 		}
 
-		process_common(attacker);
+    	process_common(attacker);
```
Comment (Yimura) [inline src/backend/reactions/interloper_reaction.cpp]: Indentation is off

**Review by Yimura — CHANGES_REQUESTED** [review]
Don't forget we have a .clang-format file, you can use it to help you format files.

Also small nitpick, why exactly is this C++23 related? At most this would just be a general code cleanup.

**Comment by MoneyDropLobby** [comment]
should be fixed now

File: src/backend/reactions/reaction.cpp [hunk 3]
```diff
@@ -64,28 +64,31 @@ namespace big
...
-		}
+            if (g_hooking->get_original<hooks::send_chat_message>()(
+                *g_pointers->m_gta.m_send_chat_ptr,
+                g_player_service->get_self()->get_net_data(),
+                chat.c_str(),
+                false))
+            {
+                notify::draw_chat(chat, g_player_service->get_self()->get_name(), false);
+            }
+        });
```
Comment (Yimura) [inline src/backend/reactions/reaction.cpp]: Indentation is still off.

File: src/backend/reactions/interloper_reaction.cpp (hunk 2 above)
Comment (Yimura) [inline src/backend/reactions/interloper_reaction.cpp]: Indentation is still off.

**Review by Yimura — CHANGES_REQUESTED** [review]
As I said before there's a formatter file in the repo, check out the repository locally and fix the indents.

Using Github's code editor fucks with indents.

**Comment by MoneyDropLobby** [comment]
I am done.

**Review by Yimura — APPROVED** [review]
No further complaints.

@MoneyDropLobby what do you use as IDE? If it is Visual Studio it should auto format the code upon save. Otherwise if it's Visual Studio Code you'll have to manually install clang-format from LLVM. You can then use an extension to format the code for you or use the below command to format all files in the `src/` directory.

```bash
clang-format -i src/**.hpp src/**.cpp
```

**Pull request merged**